src/ocm/components/clusterConfiguration/networkConfiguration/networkConfigurationValidation.ts

```diff
@@ -1,48 +1,42 @@
 import * as Yup from 'yup';
-import { Address4, Address6 } from 'ip-address';
 import {
+  allSubnetsIPv4,
   Cluster,
-  ClusterNetwork,
   clusterNetworksValidationSchema,
   dualStackValidationSchema,
+  getDefaultNetworkType,
   HostSubnets,
-  MachineNetwork,
+  isSingleStack,
+  isSNO,
   machineNetworksValidationSchema,
   NetworkConfigurationValues,
-  ServiceNetwork,
   serviceNetworkValidationSchema,
   singleStackValidationSchema,
   sshPublicKeyValidationSchema,
-  uniqueSubnetValidationSchema,
   vipValidationSchema,
 } from '../../../../common';
 
-export const isSingleStack = (
-  machineNetworks?: MachineNetwork[],
-  clusterNetworks?: ClusterNetwork[],
-  serviceNetworks?: ServiceNetwork[],
-) =>
-  (machineNetworks?.every((network) => network.cidr && Address4.isValid(network.cidr)) ||
-    machineNetworks?.every((network) => network.cidr && Address6.isValid(network.cidr))) &&
-  (clusterNetworks?.every((network) => network.cidr && Address4.isValid(network.cidr)) ||
-    clusterNetworks?.every((network) => network.cidr && Address6.isValid(network.cidr))) &&
-  (serviceNetworks?.every((network) => network.cidr && Address4.isValid(network.cidr)) ||
-    serviceNetworks?.every((network) => network.cidr && Address6.isValid(network.cidr)));
-
 export const getNetworkInitialValues = (
   cluster: Cluster,
   defaultNetworkValues: Partial<NetworkConfigurationValues>,
 ): NetworkConfigurationValues => {
+  const isSNOCluster = isSNO(cluster);
+  const usesIPv6 = !(
```

Comment:
looks like a bug... usesIpv6 checks if networks are in Ipv4... I'm pretty sure you'll see the bug if you open an existing cluster with IPV6 that wasn't assigned a network type yet (it's pretty hard to reproduce this)

Reply:
It's negated, so it should evaluate `true` if there is at least one IPv6 subnet. Since these data that come from the BE, I think we can assume they'll be valid. We cannot use `allSubnetsIPv6` because that wouldn't work with dual-stack where the IP versions are mixed.

Reply:
arghh... I keep missing the "not" operator